### PR TITLE
fix(rest-client): linting and generator config

### PR DESCRIPTION
**Pull request by MetecanAydin**
[5524](https://dev.azure.com/amido-dev/Amido-Stacks/_workitems/edit/5524)

- replaces both slashes when normalizing options
- import linting fix
- typeError for unacceptable project to bump version
- removes directory option for bump-version generator as it reads project list

### CODE REVIEW

File: packages/common/core/src/lib/normalize-options.ts [hunk 1]
```diff
@@ -39,7 +39,9 @@ export function normalizeOptions<TSchema extends BaseSchema>(
         projectDirectory,
     );
 
-    const projectName = projectDirectory.replace(/\//g, '-');
+    const projectName = projectDirectory
+        .replace(/\//g, '-')
+        .replace(/\\/g, '-');
```
Comment (CerdenAmido) [inline packages/common/core/src/lib/normalize-options.ts:44]: May be cleaner to use `path.sep` here

Comment (MetecanAydin) [reply packages/common/core/src/lib/normalize-options.ts]: updated, cheers.

**Review by nandorszentpeteri — APPROVED** [review]

**Comment by Danphillipz** [comment]
@MetecanAydin - Not sure why that E2E task failed :/

File: packages/common/core/src/lib/normalize-options.ts (hunk 1 above)
Comment (MetecanAydin) [reply packages/common/core/src/lib/normalize-options.ts:44]: change didn't work as expected. Reverted back to double replace for now.

**Pull request merged**